### PR TITLE
- Following Convention

### PR DESCRIPTION
**Pull request by Phenek**
Still Unstable

### CODE REVIEW

File: PanCardView/CoverFlowView.cs [hunk 1]
```diff
@@ -56,12 +56,86 @@ public class CoverFlowView : AbsoluteLayout
...
+            SetPanGesture();
+        }
+
+        public CoverFlowView()
+        {
+            this.ViewProcessor = new BaseCoverFlowProcessor(this);
+            this.IsClippedToBounds = true;
+
+            SetPanGesture();
+        }
```
Comment (AndreiMisiukevich) [inline PanCardView/CoverFlowView.cs]:
```csharp
public CoverFlowView(BaseCoverFlowProcessor baseCoverFlowProcessor)
        {
            _viewProcessor = baseCoverFlowProcessor;
            _viewProcessor.CoverFlow = this;
            IsClippedToBounds = true;
             SetPanGesture();
        }
         public CoverFlowView() : this (new BaseCoverFlowProcessor())
        {
        }
```
Let's skip writing "this" keyword. And we should avoid code duplicating.

File: PanCardView/CoverFlowView.cs [hunk 2]
```diff
@@ -56,12 +56,86 @@ public class CoverFlowView : AbsoluteLayout
...
+
+        /// <summary>
+        /// Gets or set the displayed views list.
+        /// </summary>
+        /// <value>The displayed views list.</value>
+        private IAbsoluteList<View> DisplayedViews => Children;
```
Comment (AndreiMisiukevich) [inline PanCardView/CoverFlowView.cs]: Redundant property. Why can't we work with Children directly ?)

File: PanCardView/CoverFlowView.cs [hunk 3]
```diff
@@ -56,12 +56,86 @@ public class CoverFlowView : AbsoluteLayout
...
+        /// Gets the maximun item index on axis.
+        /// </summary>
+        /// <value>the maximun item index on axis.</value>
+        public int ItemMaxOnAxis { get; set; } = 0;
+
+        /// <summary>
+        /// Gets the minimum item index on axis.
+        /// </summary>
+        /// <value>the minimum item index on axis.</value>
+        public int ItemMinOnAxis { get; set; } = 0;
```
Comment (AndreiMisiukevich) [inline PanCardView/CoverFlowView.cs]: All properties above - remove default 0 value, please

File: PanCardView/CoverFlowView.cs [hunk 4]
```diff
@@ -124,52 +198,107 @@ public CoverItemPosition FirstItemPosition
...
         {
+            base.OnSizeAllocated(_width, _height);
+
+            if (Math.Abs(_width - width) > 0.1 || Math.Abs(_height - height) > 0.1)
+            {
+                _isOrientationChanged = true;
+            }
+
+            if (width < 0 || height < 0 || !DisplayedViews.Any())
+                return;
```
Comment (AndreiMisiukevich) [inline PanCardView/CoverFlowView.cs]: Could you please use { } for one lines as well

```csharp
if(condition)
{
    return;
}
```

**Comment by AndreiMisiukevich** [comment]
@Phenek hi) thank you so much =)
Could you please look through comments and modify code)

File: PanCardView/CoverFlowView.cs (hunk 2 above)
Comment (Phenek) [reply PanCardView/CoverFlowView.cs]: Yeah it's redundant but more explicit,
I will keep it for my personal comfort when reading the code.

Anyway Feel free to remove it, if it's annoying you.

**Comment by Phenek** [comment]
For private readonly fileds on
```
        private List<View> _recycledViews = new List<View>();
        private BaseCoverFlowProcessor ViewProcessor { get; }
```
I can't cause of Getter and Setter, And it's like in cardsView Class.

**Comment by Phenek** [comment]
After that don't release it.
Let it obsolete, cause it's unstable.

**Comment by AndreiMisiukevich** [comment]
@Phenek sure colleague)

**Pull request merged**